dev/ci/ci-metacoq.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download metacoq
+
+( cd "${CI_BUILD_DIR}/metacoq" && ./configure.sh local && make ci-local && make install )
```

Comment:
It seems the `ci-local` target also does configure

Reply:
It shouldn't, I'll fix that upstream.